dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/statelisteners/ElasticListener.java

```diff
@@ -79,7 +80,8 @@ public class ElasticListener implements StateListenerInterface {
     public static DockstoreWebserviceConfiguration config;
     public static final String TOOLS_INDEX = "tools";
     public static final String WORKFLOWS_INDEX = "workflows";
-    public static final String ALL_INDICES = "tools,workflows";
+    public static final String NOTEBOOKS_INDEX = "notebooks";
+    public static final List<String> INDEXES = List.of(TOOLS_INDEX, WORKFLOWS_INDEX, NOTEBOOKS_INDEX);
```

Comment:
Oh, this is why. Still prefer indices, indexes sounds a bit non-technical

> Technical contexts prefer indices to indexes. The S&P Dow Jones Indices, the leading resource for investable indices, follows this form.

For example, but :probably bike-shedding